test/functional/Valhalla/src_lw5/org/openj9/test/lworld/ValueTypeArrayTests.java

```diff
@@ -0,0 +1,731 @@
+   }
+
+   @Test(priority=1,invocationCount=2)
+	static public void testValueTypeAastore() throws Throwable {
+      int ARRAY_LENGTH = 10;
+      SomeFlattenableClassWithDoubleField![] srcData1 = new SomeFlattenableClassWithDoubleField![ARRAY_LENGTH];
+      SomeFlattenableClassWithDoubleField![] dstData1 = new SomeFlattenableClassWithDoubleField![ARRAY_LENGTH];
+      SomeFlattenableClassWithFloatField![]  srcData2 = new SomeFlattenableClassWithFloatField![ARRAY_LENGTH];
+      SomeFlattenableClassWithFloatField![]  dstData2 = new SomeFlattenableClassWithFloatField![ARRAY_LENGTH];
+      SomeFlattenableClassWithLongField![]   srcData3 = new SomeFlattenableClassWithLongField![ARRAY_LENGTH];
+      SomeFlattenableClassWithLongField![]   dstData3 = new SomeFlattenableClassWithLongField![ARRAY_LENGTH];
+
+      SomeIdentityClassWithDoubleField[] srcData4  = new SomeIdentityClassWithDoubleField[ARRAY_LENGTH];
+      SomeIdentityClassWithDoubleField[] dstData4  = new SomeIdentityClassWithDoubleField[ARRAY_LENGTH];
+      SomeIdentityClassWithFloatField[]  srcData5  = new SomeIdentityClassWithFloatField[ARRAY_LENGTH];
+      SomeIdentityClassWithFloatField[]  dstData5  = new SomeIdentityClassWithFloatField[ARRAY_LENGTH];
+      SomeIdentityClassWithLongField[]   srcData6  = new SomeIdentityClassWithLongField[ARRAY_LENGTH];
+      SomeIdentityClassWithLongField[]   dstData6  = new SomeIdentityClassWithLongField[ARRAY_LENGTH];
+
+      SomeClassHolder holder   = new SomeClassHolder();
+
+      for (int i=0; i<ARRAY_LENGTH; ++i) {
+         srcData1[i] = new SomeFlattenableClassWithDoubleField((double)(i+1));
+         srcData2[i] = new SomeFlattenableClassWithFloatField((float)(i+1));
+         srcData3[i] = new SomeFlattenableClassWithLongField((long)(i+1));
+
+         dstData1[i] = new SomeFlattenableClassWithDoubleField((double)i);
+         dstData2[i] = new SomeFlattenableClassWithFloatField((float)i);
+         dstData3[i] = new SomeFlattenableClassWithLongField((long)i);
+
+         srcData4[i] = new SomeIdentityClassWithDoubleField((double)(i+1));
+         srcData5[i] = new SomeIdentityClassWithFloatField((float)(i+1));
+         srcData6[i] = new SomeIdentityClassWithLongField((long)(i+1));
+
+         dstData4[i] = new SomeIdentityClassWithDoubleField((double)i);
+         dstData5[i] = new SomeIdentityClassWithFloatField((float)i);
+         dstData6[i] = new SomeIdentityClassWithLongField((long)i);
+      }
+
+      writeArrayElementWithDoubleField(srcData1, dstData1);
+      writeArrayElementWithFloatField(srcData2, dstData2);
+      writeArrayElementWithLongField(srcData3, dstData3);
+
+      writeArrayElementWithDoubleField(srcData4, dstData4);
+      writeArrayElementWithFloatField(srcData5, dstData5);
+      writeArrayElementWithLongField(srcData6, dstData6);
+
+      writeArrayElementWithSomeFlattenableClassImplIf(holder);
+      writeArrayElementWithSomeIdentityClassImplIf(holder);
+   }
```

Comment:
It looks like this entire range of the test in both this, the `src_lw5` version, and in the `src_qtypes` version used leading spaces for indentation.  There are also some similar lines for the nested class declarations above.  May I trouble you to clean those up?

Reply:
Yes, I just pushed changes for this.